### ISSUE TITLE
English Docs

### ISSUE THREAD

**Issue by 7ammer** [LaravelRUS/SleepingOwlAdmin-docs#1]
I plan to translate the Russian docs (with the help from google translate) into English. Is there anything I should know before I proceed?

**Comment by butschster** [comment]
Hi. It's would be great! You can ask questions and i'll try to help you.

**Comment by Butochnikov** [comment]
@7ammer как с переводом, получается?

**Comment by EvgenyVetrov** [comment]
@7ammer , we started Eng version of docs

**Comment by 7ammer** [comment]
Would it help if I did a google translate of all of the .md files from branch "patch-12"? This would give us a nice starting point.

**Comment by EvgenyVetrov** [comment]
yes, you can! (maybe translate.ru better than translate.google.ru)
English version will be in /eng/ directory. My pull requests waiting that

**Comment by s6carlo** [comment]
I can help you, event I'm not English mother tongue. I'm using the sleepingowl from previous packages and it is great, but English docs is missing. The new community is mainly Russian and I am afraid I will not have english support when needed, so let's try to open the project to an English community!

**Comment by Butochnikov** [comment]
@s6carlo когда сможешь помочь с переводом?

**Comment by s6carlo** [comment]
I can contribute, but please write me in English.

**Comment by Butochnikov** [comment]
@s6carlo "... write me in English" ? but documentation in Russian )

**Comment by s6carlo** [comment]
I know documentation is in Russian, and I don't know this language. However English docs is missing, and I think is better to try to translate using a translator and testing the result than having an empty documentation.  Of course if there are some Russian and English speakers available to translate docs is better

**Comment by Butochnikov** [comment]
https://github.com/translation-gang/en.docs.sleeping-owl-admin
